src/main/java/uk/gov/hmcts/dts/fact/controllers/SearchController.java

```diff
@@ -48,6 +47,11 @@ public SearchController(final CourtService courtService) {
         @RequestParam(required = false, name = "q") Optional<String> query
     ) {
         if (postcode.isPresent() && areaOfLaw.isPresent()) {
+            switch(areaOfLaw.get().toLowerCase()) {
```

Comment:
If there is only a single case in this condition, I think it is probably more conventional to use if statement rather than the switch statement :)

Reply:
Hi @KianKwa - fully agree, I expected to implement a load of different AOLs here but luckily realised we can get away with one, just forgot to go change this to something more appropriate. Re: the change to lower-case, I don't see this done elsewhere in the system where AOL has been taken into account, hence intentionally leaving it out (I thought the same as you at first). Might be worth ensuring consistency in how it's handled across the API and then potentially re-addressing this in the future.